pegasus/test/test_csv.rb

```diff
@@ -0,0 +1,15 @@
+require_relative '../../lib/cdo/pegasus'
+require 'minitest/autorun'
+
+class CSVTest < Minitest::Test
+  def test_cdo_donors_csv
+    donors_csv = CSV.read('./data/cdo-donors.csv')
+    max_donor_weight = donors_csv.last[4]
+    assert_equal max_donor_weight.to_f, 1.0
+  end
+
+  def test_cdo_donors_db
+    donors_with_max_weight = DB[:cdo_donors].where(weight_f: 1.0)
+    assert_equal donors_with_max_weight.count, 1
```

Comment:
Precisely; we do a weighted select using that field on a randomly-generated number from 0 to 1.0. We've had a few errors where the top entry had a weight of < 1.0, so the randomly-generated number ended up selecting nothing for the weighted random selection.